pom.xml

```diff
@@ -426,7 +426,7 @@
 
 					<!-- Third argument prevents Java from popping up lots of windows on
 						MacOS -->
```

Comment:
no, please leave it at 2G as there are a couple of tests are designed for 2G which would otherwise fail with an unexpected number of compiled/executed distributed operations.